db_helper.php

```diff
@@ -1,98 +1,129 @@
 <?php
+if (!file_exists('config.php')) {
+    exit('Please copy \'config.sample.php\' to \'config.php\'');
+}
+require_once 'config.php';
```

Comment:
I'm just curious: Why is this required here but not in the other files?

Reply:
All other files include `db_helper.php`, so the check runs every time.

Reply:
Meh, relying on assumptions about which files include which other files is something I would like to avoid. Just remove the file_exists check please.

Reply:
Okay